Documentation/ceph-csi-drivers.md

```diff
@@ -121,10 +121,13 @@ A volume claim template is defined inside the pod spec which refers to a volume
 provisioned and used by the pod with its lifecycle. The volumes are provisioned
 when pod get spawned and destroyed at time of pod delete.
 
-Refer to [ephemeral-doc]( https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/#generic-ephemeral-volumes )
-for more info. Also, See the example manifests for an [RBD ephemeral volume]
-(https://github.com/rook/rook/tree/{{ branchName }}/cluster/examples/kubernetes/ceph/csi/rbd/pod-ephemeral.yaml)
-and a [CephFS ephemeral volume](https://github.com/rook/rook/tree/{{ branchName }}/cluster/examples/kubernetes/ceph/csi/cephfs/pod-ephemeral.yaml).
+Refer to [ephemeral-doc](
```

Comment:
The change needed is that links must be on the same line as their link text. The links cannot span lines or else the renderer gets confused. For example:
```
[RBD ephemeral volume](https://github.com/rook/rook/tree/{{ branchName }}/cluster/examples/kubernetes/ceph/csi/rbd/pod-ephemeral.yaml)
```

Reply:
changed to a single line.

Reply:
even if a add like below in my vi editor. this is getting moved to a new line as line length is more

Reply:
Yep, the default line counter setting in the IDE :) ,